LIPS/lip-3.md

```diff
@@ -0,0 +1,152 @@
+
+# Summary
+
+## Problem
+
+Lido DAO governance currently relies on Aragon Voting App model. This means DAO approves or rejects proposals via direct governance token voting. Though transparent and reliable, it is not a convenient way to make decisions only affecting small groups of Lido DAO members. Besides, direct token voting doesn't exactly reflect all the decision-making processes within the Lido DAO and is often used only to formalize an existing consensus.
+There are a few natural sub-governance groups within the DAO, e.g. node operators committee, financial operations team and LEGO committee. Every day they need to take routine actions only related to their field of expertise. The decisions they make hardly ever spark any debate in the community, and votings on such decisions often struggle to attract wider DAO attention and thus, to pass.
+
+## Proposed solution
+
+We propose Easy Track frictionless motions as a solution to this problem. 
+Easy Track motion is a lightweight voting considered to have passed if the minimum objections threshold hasn’t been exceeded. As opposed to traditional Aragon votings, Easy Track motions are cheaper (no need to vote ‘pro’, token holders only have to vote ‘contra’ if they have objections) and easier to manage (no need to ask broad DAO community vote on proposals sparking no debate). 
+
+## Use cases for Easy Track motions
+
+There are three types of votings run periodically by the Lido DAO that are proposed to be wrapped into the new Easy Track motions:
+- Node Operators increasing staking limits
+- Funds being allocated into reward programs
+- Funds being allocated to LEGO program
+
+More types of motions can be added later, e.g.:
+- Topping up Lido staking referral program
+- Purchasing staking cover
+
+## Sub-governance groups
+
+There's a sub-governance group within the Lido DAO for each of the voting cases listed above. Those groups have been formed intentionally or in a more natural way, and at this point creating and maintaining periodical Aragon votings falls within their remits.
+The proposed improvement will require formalizing sub-governance groups:
+
+1. **Node Operators Committee** to start staking limit increase motions.
+2. **Financial Operations Team** to start funds allocation motions (i.e. LEGO and reward programs).
+
+Node Operators Committee should include addresses of active node operators partnering with Lido, the list is easy to look up in the [Node Operators Registry](https://github.com/lidofinance/lido-dao/blob/master/contracts/0.4.24/nos/NodeOperatorsRegistry.sol) smart contract. Financial Operations Team should consist of established DAO members' addresses and be represented as a Gnosis Safe Multisig with access to corresponding Easy Track features.
+
+# Specification
+
+To create a feature as flexible and extendable as possible, we propose implementing Easy Track functionality within several contracts rather than building a single contract for it. 
```

Comment:
Like I put it in the LIP, flexibility and extendability were the two primary reasons why we preferred splitting the feature into multiple contracts rather than building a Frankenstein master contract instead. Another reason worth mentioning is having the feature supervised, adding it into the proposal.